### PR TITLE
Fix requirements, add contributors, pin version.

### PR DESCRIPTION
**Pull request by somexpert**
ping @gracieflores, @ldko, this is ready for review.

### CODE REVIEW

**Comment by gracieflores** [comment]
The changes look good and thanks for fixing what I missed. 👍🏼

**Review by gracieflores — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ x.x.x
 * Dropped support for Python 2.7 and 3.4-3.5.
 * Added Python 3.8 and 3.9 testing.
 * Upgraded Flask to 2.0.3.
+* Update the list of contributors
```
Comment (ldko) [inline CHANGELOG.md]: Personally, I wouldn't include this in the CHANGELOG.md, but if you think we should, I would change "Update" to "Updated" for consistency. Also for consistency, I would add periods at the end of the added lines.

**Review by ldko — COMMENTED** [review]
Consider the comment about the CHANGELOG.md, but otherwise, I approve :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (somexpert) [reply CHANGELOG.md]: I've removed it. I wasn't sure about adding it in the first place :smile:

**Pull request merged**